components/logins/android/src/main/java/mozilla/appservices/logins/ImportMetrics.kt

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.appservices.logins
+import org.json.JSONArray
+// import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Raw password data that is stored by the LoginsStorage implementation.
+ */
+data class ImportMetrics(
+
+    /**
+     * Total duration of the import in ms
+     */
+   val totalDuration: Int,
+
+    /**
+     * List of metrics for each import phase
+     */
+    val phases: List<ImportPhaseMetrics>,
+
+    /**
+     * Number of records that failed during the import
+     */
+    val numFailed: Int,
```

Comment:
This was previously a `Long`, does it matter that it's now an `Int`. I can't imagine us overflowing this in practice so I guess not..

Reply:
I also wonder if it would be worth reporting a top-level `numSucceeded` and/or `numProcessed` count here for completeness, rather than having to dig that out of the `phases` list.

Reply:
Good catch on the Int/Long. I can change it back. I can easily add those two metrics.

Reply:
Created  #2407 to capture this work. I'll tag it for the migration once it makes it's way to JIRA.